README.md

```diff
@@ -436,6 +436,16 @@ def result = invokeLambda(
 )
 ```
 
+Alternatively payload and return value can be Strings instead of Objects:
+
+```
+String result = invokeLambda(
+	functionName: 'myLambdaFunction',
+	payload: '{"key": "value"}',
```

Comment:
the implementation says `payloadAsString`

Reply:
Indeed! I will fix that.